ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -276,10 +273,13 @@ export default class TaskRunsCard extends Component {
                   classes.linkCell,
                   classes.artifactsTableCell
                 )}>
+            <Link
+              className={classes.artifactLink}
+              to={this.getArtifactUrl(artifact)}>
```

Comment:
Clicking on an artifact doesn't seem to work anymore. Are you able to replicate?

Reply:
I am able to fix the `validateDomNesting()..` issue and yes, clicking on artifact works, but now I get a series of prettier/prettier warnings because of which my tests fail when I push to the repo.


![Untitled](https://user-images.githubusercontent.com/30138596/66600817-09ad9900-ebc4-11e9-8183-fc5bbbade838.png)

Reply:
Yes, I think I fixed it, squashed all the previous commits and I hope it passes all the check this time. Fingers crossed,
![Untitled](https://user-images.githubusercontent.com/30138596/66601644-087d6b80-ebc6-11e9-9029-d935cad39452.png)

Reply:
Tests pass :)